prow/plank/plank_test.go

```diff
@@ -237,6 +238,7 @@ func TestSyncJenkinsJob(t *testing.T) {
 					CompletionTime: time.Now(),
 				},
 			},
+			pendingJobs:      map[string][]kube.ProwJob{},
```

Comment:
By "it," dp you mean the the loop that runs the test case? The current implementation requires that the `sync{Kubernetes|Jenkings}Job` functions have a non-nil map in order to append jobs which become `Pending` to it...might be a better way to do this tho..